From fd448cb1e28727aa845b50350d755bf4b14322ed Mon Sep 17 00:00:00 2001
From: Nicolas Goaziou <mail@nicolasgoaziou.fr>
Date: Mon, 28 Jul 2014 15:14:53 +0200
Subject: [PATCH 04/26] Remove some options associated to variables

* lisp/ox-html.el: Remove options.

* lisp/ox-icalendar.el: Remove options.

* lisp/ox-latex.el (org-latex-guess-inputenc, org-latex-compile,
  org-latex--collect-warnings): Use full variable name instead of an
  option.

* lisp/ox-man.el: Remove options.

* lisp/ox-odt.el: Remove options.

* lisp/ox-texinfo.el: Remove options.

* doc/org.texi (Publishing options): Update documentation accordingly.

These variables are meant to be used outside (or on the fringe) of the
export process, where information channel is not available.

remove
---
 lisp/ox-odt.el | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/lisp/ox-odt.el b/lisp/ox-odt.el
index 5500ebf3e..b6f237c42 100644
--- a/lisp/ox-odt.el
+++ b/lisp/ox-odt.el
@@ -98,11 +98,6 @@
   '((:odt-styles-file "ODT_STYLES_FILE" nil nil t)
     ;; Other variables.
     (:odt-content-template-file nil nil org-odt-content-template-file)
-    (:odt-convert-capabilities nil nil org-odt-convert-capabilities)
-    (:odt-convert-process nil nil org-odt-convert-process)
-    (:odt-convert-processes nil nil org-odt-convert-processes)
-    (:odt-create-custom-styles-for-srcblocks
-     nil nil org-odt-create-custom-styles-for-srcblocks)
     (:odt-display-outline-level nil nil org-odt-display-outline-level)
     (:odt-fontify-srcblocks nil nil org-odt-fontify-srcblocks)
     (:odt-format-drawer-function nil nil org-odt-format-drawer-function)
@@ -111,9 +106,6 @@
     (:odt-inline-formula-rules nil nil org-odt-inline-formula-rules)
     (:odt-inline-image-rules nil nil org-odt-inline-image-rules)
     (:odt-pixels-per-inch nil nil org-odt-pixels-per-inch)
-    (:odt-preferred-output-format nil nil org-odt-preferred-output-format)
-    (:odt-prettify-xml nil nil org-odt-prettify-xml)
-    (:odt-schema-dir nil nil org-odt-schema-dir)
     (:odt-styles-file nil nil org-odt-styles-file)
     (:odt-table-styles nil nil org-odt-table-styles)
     (:odt-use-date-fields nil nil org-odt-use-date-fields)
-- 
2.13.2

